api/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7
+
+LABEL Author="Brylie Christopher Oxley"
+LABEL E-mail="brylie.oxley@maas.global"
+LABEL version="0.0.1"
+
+LABEL Name=sustainable-mobility-api Version=0.0.1
+EXPOSE 8080
+
+WORKDIR /app
+ADD . /app
+
+RUN python -m pip install pipenv
+RUN pipenv install --ignore-pipfile
+CMD ["pipenv", "run", "python3", "-m", "api"]
```

Comment:
We live in a Python 3 world here, so why not default to `python` and forget, Python 2 and that whole transition mess ever existed? 😉